puppet/modules/quickstack/manifests/params.pp

```diff
@@ -58,9 +58,9 @@
   $cinder_dell_sc_san_login        = ['Admin'],
   $cinder_dell_sc_san_password     = ['CHANGEME'],
   $cinder_dell_sc_iscsi_ip_address = ['192.168.0.20'],
-  $cinder_dell_sc_iscsi_port       = [3260],
+  $cinder_dell_sc_iscsi_port       = ['3260'],
```

Comment:
I didn't realize that they aren't used anymore. Looks like these files are not deprecated and I was asked not to update them. Some cleanup required?

puppet/modules/quickstack/manifests/controller_common.pp
puppet/modules/quickstack/manifests/neutron/controller.pp
puppet/modules/quickstack/manifests/nova_network/controller.pp